CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+Version 0.10.1 (October 1?, 2021)
+---------------------------------
+
+Small bug-fix release:
+
+- Fix regression in `overlay()` in case no geometries are intersecting (but
+  have overlapping total bounds) (#2172).
+- Fix regression in `overlay()` with `keep_geom_type=True` in case the
+  overlay of two geometries in a GeometryCollection with other geometry types
+  (#2177).
+- Fix `overlay()` to honor the `keep_geom_type` keyword for all overlay
+  methods (i.e. not only for the default `op="intersection"`) (#2164).
```

Comment:
`op="difference"` was the only one that wasn't going through the `keep_geom_type` logic, but perhaps that's not an especially important distinction to make.

Reply:
Ah, yes. Updated.